triggers/tests/src/test/scala/com/digitalasset/daml/lf/engine/trigger/test/DevOnly.scala

```diff
@@ -16,6 +16,7 @@ import org.scalatest.matchers.should.Matchers
 import org.scalatest.wordspec.AsyncWordSpec
 import com.daml.lf.engine.trigger.TriggerMsg
 import com.daml.util.Ctx
+import scalaz.syntax.tag._
```

Comment:
Changes in this file all relate to using `Party` rather than `String` as a method argument